### DISCUSSION TITLE
Add multiple (more) Analytics and Comment system support?

### DISCUSSION THREAD

**Discussion by Zenuncl** [adityatelange/hugo-PaperMod#783] [Ideas]
Such as adding following Analytics system and Commenting system to the theme?

Analytics
  - Fathom Analytics
  - Plausible Analytics
  - Cloudflare
  - Matomo

Commenting Systems
  - Commento
  - Utterances

**Comment by LITUATUI** [comment]
I would prefer [Giscus](https://giscus.vercel.app/), since it uses the new Discussions section instead of Issues for comments.

**Comment by silent1mezzo** [comment]
👍 for Fathom Analytics
